src/model_version_policy.cpp

```diff
@@ -38,6 +44,12 @@ std::vector<model_version_t> SpecificModelVersionPolicy::filter(std::vector<mode
     return result;
 }
 
+SpecificModelVersionPolicy::operator std::string() const {
+    std::stringstream versionStream;
+    std::copy(specificVersions.begin(), specificVersions.end(), std::ostream_iterator<model_version_t>(versionStream, " "));
+    return std::string("specific: ") + versionStream.str();
```

Comment:
in line 50 you create string(specific:), then create second string(versionStream.str()), then call operator+(lhs,rhs) which creates third string.

versionStream << "specific: ";
then return versionStream.str().